composer.json

```diff
@@ -2,12 +2,13 @@
   "name": "code-tool/doctrine-dbal-jaeger",
   "license": "MIT",
   "require": {
-    "php": "^7.4 || ^8",
-    "doctrine/dbal": "^3.0",
+    "php": "^8.1",
+    "doctrine/dbal": "^4.2",
     "code-tool/jaeger-client-php": "^3"
-  },
+  }
+  ,
   "require-dev": {
-    "phpunit/phpunit": "^7.2 || ^8"
+    "roave/security-advisories": "dev-latest"
```

Comment:
why is it needed?

Reply:
As for me, require this package is best practice. One can forgot to call `composer audit`, but this package will not allow to use non-secure package.

Reply:
OK, let's keep it. Thank you